rootfs/bin/boot

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+echo "Powered by $POWERED_BY" > /usr/share/nginx/html/index.html
```

Comment:
Of course.  Sorry.  My eyes did an automatic substitution of "Deis" for `$POWERED_BY `, so I wasn't even thinking about that.